src/bootstrap/native.rs

```diff
@@ -171,7 +171,6 @@ impl Step for Llvm {
             .define("LLVM_TARGETS_TO_BUILD", llvm_targets)
             .define("LLVM_EXPERIMENTAL_TARGETS_TO_BUILD", llvm_exp_targets)
             .define("LLVM_INCLUDE_EXAMPLES", "OFF")
-            .define("LLVM_INCLUDE_TESTS", "OFF")
```

Comment:
Is this necessary still? I doubt this hurts much, since all it does is adding more targets to ninja, targets that ultimately go unused anyway. Or at least that should be the case.

Reply:
Yes, it's necessary to re-enable tests. See 'Enable building LLVM tests' in the PR description.

Reply:
Yeah, I am a bit worried about this, but we can revert this PR if it becomes a problem I think.